docs/releases.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+## 1.2.0-beta3
+Fixes part of this release:
+ 1. TestMethod failures masked by TestCleanUp exceptions. [#58](https://github.com/Microsoft/testfx/issues/58)
+ 2. Multiple copies added for same test on running multiple times in IntelliTest. [#92](https://github.com/Microsoft/testfx/issues/92)
+ 3. Adapter is not sending TestCategory traits in Testcase object to Testhost. [#189](https://github.com/Microsoft/testfx/issues/189)
+ 4. Added Mapping for TestOutcome.None to the UnitTestOutcome Enum to achieve NotExecuted behaviour in VSTS. [#217](https://github.com/Microsoft/testfx/issues/217) [enhancement]

```

Comment:
Please move the enhancement on top and tag external contributors as well please. See below for context.